.azure/choreo-keda-http-addon-interceptor-build.yaml

```
@@ -57,3 +70,17 @@ steps:
       tags: |
         latest
         $(TAG)
+  - task: Docker@2
+    displayName: "Push Docker image"
+    inputs:
+      command: push
+      containerRegistry: 'choreo-private-acr'
+      repository: $(REPOSITORY)
+      tags: |
+        latest
+        $(TAG)
+  - template: update-image-in-cp-overlays.yml@common-templates
+    parameters:
+      CLUSTER_IMAGE_MAP:
+        privatedataplane:
+          - ${{ variables.PDP_CONTAINER_REGISTRY }}/${{ variables.AGENT_REPOSITORY }}:${{ variables.APP_REVISION }}
```

```suggestion
          - $(PDP_CONTAINER_REGISTRY)/$(AGENT_REPOSITORY):$(APP_REVISION)
```

use this syntax for referring to variables in all the places